crates/store/re_query/src/latest_at.rs

```diff
@@ -39,7 +39,7 @@ fn compare_indices(lhs: (TimeInt, RowId), rhs: (TimeInt, RowId)) -> std::cmp::Or
 }
 
 impl QueryCache {
-    /// Queries for the given `component_names` using latest-at semantics.
+    /// Queries for the given [`ComponentDescriptor`]s using latest-at semantics.
```

Comment:
not entiiiiirely true yet. it takes everything that intos into a descriptor and then only uses the name for now